repo/packages/Q/quobyte/0/marathon.json

```diff
@@ -0,0 +1,42 @@
+{
+  "id": {{quobyte.framework-name}},
+  "cpus": "1.0",
+  "mem": "1024",
+  "instances": "1",
```

Comment:
Ok, will add that. 
Probably a good idea to add this to the readme, too, it currently shows this data as example.